dateutil/test/test_relativedelta.py

```diff
@@ -303,6 +303,41 @@ def testComparison(self):
         self.assertEqual(d1, d2)
         self.assertNotEqual(d1, d3)
 
+    def testNotImplementedComparison(self):
+        with self.assertRaises(TypeError):
+            relativedelta(month=1) > relativedelta(days=30)
+
+        with self.assertRaises(TypeError):
+            relativedelta(month=1) >= relativedelta(days=30)
+
+        with self.assertRaises(TypeError):
+            relativedelta(month=1) < relativedelta(days=30)
+
+        with self.assertRaises(TypeError):
+            relativedelta(month=1) <= relativedelta(days=30)
+
+    def textCustomComparison(self):
+        if PY2:
```

Comment:
Generally speaking, for tests that fail on Python 3 for compatibility reasons, we mark the test as skipped on Python 3 with the `skipif` marker, like so:

`@pytest.mark.skipif(six.PY3, reason="Whatever reason")`

But in this case, these tests should pass regardless, so you can just remove the `if PY2` block here.